src/AWS.Deploy.CLI/AWS.Deploy.CLI.csproj

```diff
@@ -47,6 +47,10 @@
     <None Include="..\..\README.md" Pack="true" PackagePath="" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Nerdbank.GitVersioning" Version="3.6.133" />
```

Comment:
Curious, why this addition here now?

Reply:
This was a miss. While using the Nuget Package Manager, it added it to the csproj instead of the `Directory.Build.props` file. Good catch.